README.md

```diff
@@ -90,25 +91,19 @@ $ make binary-cross
 `kompose` is a work in progress, we will see how far it takes us. We welcome any pull request to make it even better.
 If you find any issues, please [file it](https://github.com/kubernetes-incubator/kompose/issues).
 
-## Community, discussion, contribution, and support
-
-We follow the Kubernetes community principles.
-
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
-
-You can reach the maintainers of this project at:
+## Community, Discussion, Contribution, and Support
 
-- [Slack](http://slack.kubernetes.io): #kompose
+As part of the Kubernetes ecosystem, we follow the Kubernetes community principles. More information can be found on the [community page](http://kubernetes.io/community/).
 
-Incubation of `kompose` into the Kubernetes project is shepherd in the [SIG-APPS](https://github.com/kubernetes/community).
+You can reach the maintainers of this project on [Slack](http://slack.kubernetes.io) in channel #kompose
```

Comment:
My only concern is that it should not sound like "not welcoming for everyone to hangout on #kompose channel".
Whatever does not sound like above is good for me.